cmd/virt-handler/virt-handler.go

```diff
@@ -303,6 +308,7 @@ func (app *virtHandlerApp) Run() {
 	// set log verbosity
 	app.clusterConfig.SetConfigModifiedCallback(app.shouldChangeLogVerbosity)
 	app.clusterConfig.SetConfigModifiedCallback(app.shouldChangeRateLimiter)
+	app.clusterConfig.SetConfigModifiedCallback(app.shouldInstallSELinuxPolicy)
```

Comment:
Yes, `SetConfigModifiedCallback()` calls the callback once right away. In the functional test, I thought it'd be best to avoid murdering all of the existing virt-handler pods just to test that `SetConfigModifiedCallback()` does its job...